neopixel.py

```diff
@@ -342,7 +345,18 @@ def show(self):
         if self.W_in_mode:
             cut = 0
         
-        self.sm.put(self.pixels, cut)
+        data = array.array('I',self.pixels)
+        for i,_ in enumerate(data):
+            data[i] <<= cut
```

Comment:
I think this should also be merged with the option above where cut is 0, so we avoid unnecessary copy in case of RGBW. Or is this necessary?